lib/ansible/modules/cloud/google/gce_snapshot.py

```diff
@@ -0,0 +1,195 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.gce import gce_connect
+
+DOCUMENTATION = '''
+---
+module: gce_snapshot
+version_added: "2.3"
+short_description: Create or destroy snapshots for GCE storage volumes
+description:
+  - Manages snapshots for GCE instances. This module manages snapshots for
+    the storage volumes of a GCE compute instance. If there are multiple
+    volumes, each snapshot will be prepended with the disk name
+options:
+  instance_name:
+    description:
+      - The GCE instance to snapshot
+    required: True
+  snapshot_name:
+    description:
+      - The name of the snapshot to manage
+  disks:
+    description:
+      - A list of disks to create snapshots for. If none is provided,
+        all of the volumes will be snapshotted
+    default: all
+    required: False
+  state:
+    description:
+      - Whether a snapshot should be C(present) or C(absent)
+    required: false
+    default: present
+    choices: [present, absent]
+  service_account_email:
+    description:
+      - GCP service account email for the project where the instance resides
+    required: true
+  credentials_file:
+    description:
+      - The path to the credentials file associated with the service account
+    required: true
+  project_id:
+    description:
+      - The GCP project ID to use
+    required: true
+requirements:
+    - "python >= 2.6"
+    - "apache-libcloud >= 0.19.0"
+author: Rob Wagner (@robwagner33)
+'''
+
+EXAMPLES = '''
+- name: Create gce snapshot
+  gce_snapshot:
+    instance_name: example-instance
+    snapshot_name: example-snapshot
+    state: present
+    service_account_email: project_name@appspot.gserviceaccount.com
+    credentials_file: /path/to/credentials
+    project_id: project_name
+  delegate_to: localhost
+
+- name: Delete gce snapshot
+  gce_snapshot:
+    instance_name: example-instance
+    snapshot_name: example-snapshot
+    state: absent
+    service_account_email: project_name@appspot.gserviceaccount.com
+    credentials_file: /path/to/credentials
+    project_id: project_name
+  delegate_to: localhost
+
+# This example creates snapshots for only two of the available disks as
+# disk0-example-snapshot and disk1-example-snapshot
+- name: Create snapshots of specific disks
+  gce_snapshot:
+    instance_name: example-instance
+    snapshot_name: example-snapshot
+    state: present
+    disks:
+      - disk0
+      - disk1
+    service_account_email: project_name@appspot.gserviceaccount.com
+    credentials_file: /path/to/credentials
+    project_id: project_name
+  delegate_to: localhost
+'''
+
```

Comment:
Could you add a RETURN block?  You can find an example [here](https://github.com/ansible/ansible/blob/42e63d429c63e078e406d95edebd8f86cd44cfbd/lib/ansible/modules/cloud/google/gce_mig.py#L180)